specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/stable/2019-04-01/resourcegraph.json

```diff
@@ -375,7 +387,7 @@
         },
         "data": {
           "description": "A table containing the desired facets. Only present if the facet is valid.",
-          "$ref": "#/definitions/Table"
+          "type": "object"
```

Comment:
why is the type changing?

Reply:
Same as above, we added new result format, which depends from query param ResultFormat